webhooks/ssp_webhook.go

```diff
@@ -174,6 +183,21 @@ func validateDataImportCronTemplates(ssp *sspv1beta1.SSP) error {
 	return nil
 }
 
+func validateCommonInstancetypes(ssp *sspv1beta1.SSP) error {
+	if ssp.Spec.CommonInstancetypes == nil || ssp.Spec.CommonInstancetypes.URL == nil {
+		return nil
+	}
+
+	url := *ssp.Spec.CommonInstancetypes.URL
+	if !strings.HasPrefix(url, "https://") && !strings.HasPrefix(url, "ssh://") {
```

Comment:
Can/should we support `http://` too?

Reply:
The krusty/kustomize docs only highlight https so that's all I'm including for now.

Reply:
Agreed.